src/data/python-quiz.ts

```diff
@@ -1632,6 +1632,16 @@ const pythonQuiz = [
     Explanation:
       "In Python, lists have several built-in methods to perform various operations. However, the method 'add()' is not a valid method for lists.",
     Link: "https://www.freecodecamp.org/news/lists-in-python-comprehensive-guide/"
+  },
+  {
+    Question: "Which of the following statements is true regarding the Python random library?",
+    Answer: "The random library can only generate pseudo-random numbers.",
+    Distractor1: "The random library can generate true random numbers based on hardware entropy.",
+    Distractor2: "The random library is primarily used for generating random strings.",
+    Distractor3: "The random library cannot be used to generate random numbers within a specified range.",
+    Explanation:
+      "The random module in Python generates pseudo-random numbers, which are generated by mathematical algorithms and are not truly random but are sufficiently unpredictable for many purposes. To generate truly random numbers, you may need specialized hardware or external services, which are not provided by the random module.",
+    Link: "https://www.geeksforgeeks.org/python-random-module/"
```

Comment:
we only accept links from freeCodeCamp resources, wikipedia or official docs

```suggestion
    Link: "https://docs.python.org/3/library/random.html"
```